server/scripts/import-tests/populate-test-data.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const db = require('../../models');
+
+const populateTestDatabase = async () => {
+    const testDataScript = fs.readFileSync(
+        __dirname + '/pg_dump_2021_05_test_data.sql',
+        'utf-8'
+    );
+    console.info('test.data.dump', testDataScript);
+    return await db.sequelize.query(testDataScript);
+};
+
+if (require.main === module)
```

Comment:
Could you assign `require.main === module` to a variable like `isCliEnvironment` in case someone (me 🤪) doesn't know what it means?

Reply:
Yea, could be done. The plan is to move all these environment config level settings through a single module (`process.env` variables, etc) so that's coming

Reply:
Ok, no need to comment if it's temporary code.